src/ResourceTemplate/RelativeResourceTemplate.php

```diff
@@ -272,12 +272,15 @@ public function match($path)
         return $collapsedBindings;
     }
 
-    private function matchException($path, $reason)
+    /**
+     * @param string|null $path
+     */
+    private function matchException($path, string $reason)
```

Comment:
Can we add `string $path = null` here?

Reply:
We can't because declaring an optional parameter before a required parameter throws a warning in PHP 8.0

Reply:
I was able to remove support for `|null`, since this method is private.

Reply:
Cool thanks